curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145e6eeaa66c605eb087fe9.md

```diff
@@ -9,40 +9,32 @@ dashedName: step-30
 
 Although not required for `label` elements with a nested `input`, it is still best-practice to explicitly link a `label` with its corresponding `input` element.
```

Comment:
I think we should move this line to step 31.

Reply:
We could, but I thought it would be a good idea to give the user some context as to why they are linking the labels and inputs, no?

Reply:
Right, but they aren't doing the linking until step 31 now. In step 30 they are adding ids to the inputs.

Reply:
Right, okay. I'll make the changes